src/main/resources/templates/fragments/download-and-preview.html

```diff
@@ -2,21 +2,17 @@
 <html xmlns:th="http://www.thymeleaf.org" xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">
   <div th:fragment="download-and-preview(records_or_entries_string)">
     <p class="data-representations">
+      <th:block th:if="${pagination.totalEntries} le 5000">Get this data as</th:block>
+      <th:block th:if="${pagination.totalEntries} gt 5000">Get the first 5,000 records as</th:block>
+      <a th:href="${'/' + records_or_entries_string + '.json'}">JSON</a>,
+      <a th:href="${'/' + records_or_entries_string + '.yaml'}">YAML</a>,
+      <a th:href="${'/' + records_or_entries_string + '.ttl'}">TTL</a>,
       <span data-click-events="" data-click-category="Content" data-click-action="Register Download">
-        <th:block th:if="${pagination.totalEntries} le 5000">Download a copy of this register as</th:block>
-        <th:block th:if="${pagination.totalEntries} gt 5000">Download the first 5,000 records as</th:block>
         <a th:href="${'/' + records_or_entries_string + '.csv?page-size=5000'}" rel="alternate" class="js-download"
-          data-download-type="CSV" download="">CSV</a>,
+          data-download-type="CSV" download="">CSV</a> or
         <a th:href="${'/' + records_or_entries_string + '.tsv?page-size=5000'}" rel="alternate" class="js-download"
```

Comment:
I think we need to remove the page-size for CSV and TSV here to make it consistent with the other formats?